src/main/resources/syslog_logs.avro

```diff
@@ -0,0 +1,219 @@
+                "arg.properties": {
+                    "range": {
+                        "min": 0,
+                        "max": 8
+                    }
+                }
+            }
+        },
+        {
+            "name": "appName",
+            "type": {
+                "type": "string",
+                "arg.properties": {
+                    "regex": "[A-Z]{5,8}"
+                }
+            }
+        },
+        {
+            "name": "remoteAddress",
+            "type": {
+                "type": "string",
+                "arg.properties": {
+                    "options": [
+                        "91.253.222.9",
+                        "48.92.8.255",
+                        "107.150.81.0",
+                        "254.11.108.139",
+                        "14.53.111.201",
+                        "215.70.232.123",
+                        "122.96.193.183",
+                        "185.128.89.135",
+                        "246.46.59.238",
+                        "238.158.147.172"
+                    ]
+                }
+            }
+        },
+        {
+            "name": "rawMessage",
+            "type": {
+                "type": "string",
+                "arg.properties": {
+                    "regex": "[a-z ]{10,15}"
+                }
+            }
+        },
+        {
+            "name": "processId",
+            "type": {
+                "type": "string",
+                "arg.properties": {
+                    "regex": "[1-9]{1}[0-9]{6}"
+                }
+            }
+        },
+        {
+            "name": "messageId",
+            "type": {
+                "type": "string",
+                "arg.properties": {
+                    "regex": "[1-9]{1}[0-9]{2}"
+                }
+            }
+        },
+        {
+            "name": "deviceVendor",
+            "type": {
+                "type": "string",
+                "arg.properties": {
+                    "regex": "[a-z]{5}"
+                }
+            }
+        },
+        {
+            "name": "deviceProduct",
+            "type": {
+                "type": "string",
+                "arg.properties": {
+                    "regex": "[a-z]{5}"
+                }
+            }
+        },
+        {
+            "name": "deviceVersion",
+            "type": {
+                "type": "string",
+                "arg.properties": {
+                    "options": [
+                        "1.0",
+                        "2.0",
+                        "3.0",
+                        "4.0",
+                        "5.0"
+                    ]
+                }
+            }
+        },
+        {
+            "name": "ts",
+            "type": {
+                "type": "long",
+                "arg.properties": {
+                    "range": {
+                        "min": 1609459200000,
+                        "max": 1640995199000
+                    }
+                }
+            }
+        }
```

Comment:
I think I missed this before but how about doing timestamp like the others, but with a smaller step since it's syslogs, e.g.:
```
        {
            "name": "ts",
            "type": {
                "type": "long",
                "logicalType": "timestamp-millis",
                "arg.properties": {
                    "iteration": {
                        "start": 1609459200000,
                        "step": 100
                    }
                }
            }
        }
```